frontend/src/MarketplacePage/index.jsx

```diff
@@ -14,6 +15,16 @@ const MarketplacePage = ({ darkMode, switchDarkMode }) => {
   const [fetchingInstalledPlugins, setFetching] = React.useState(false);
 
   const ENABLE_MARKETPLACE_DEV_MODE = config.ENABLE_MARKETPLACE_DEV_MODE === 'true';
+  const { admin } = JSON.parse(localStorage.getItem('currentUser')) || {};
```

Comment:
this is to be changed when workspace id scoping PR is merged, will merge workspace id scoping PR first

@shah21 need your help here

Reply:
@gsmithun4 okay 🙌🏻

Reply:
@arpitnath,  you can change `const { admin } = JSON.parse(localStorage.getItem('currentUser')) || {};` to
`const { admin } = authenticationService.currentSessionValue;` when workspace id branch is merged.

Reply:
@shah21 okay 👍